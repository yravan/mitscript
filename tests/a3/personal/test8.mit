a = 5 * 5 * 5 / 5 / 5 * 5;
print(a);

a = (5 + 5) * 5;
print(a);

a = 5 * 5 / 5 / 5 / 5 / 5 * 5 * 5;
print(a);

a = 5 * (5 + 5);
print(a);

a = 5 - 5 - 5 - 5 - 5 * 5;
print(a);

a = -(-(-(5)));
print(a);

a = -5 / 5;
print(a);

a = 100 / 10 / 1 / 1 / 1 / 1;
print(a);

a = 5 * 5 / 10;
print(a);

a = 5 * (5 / 10);
print(a);