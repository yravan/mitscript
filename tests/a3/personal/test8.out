25
50
0
50
-35
-5
-1
10
2
0
